worldedit-core/src/main/java/com/sk89q/worldedit/regions/shape/WorldEditExpressionEnvironment.java

```diff
@@ -28,6 +28,8 @@
 
 public class WorldEditExpressionEnvironment implements ExpressionEnvironment {
 
+    private final Vector3 BLOCK_CENTER_OFFSET = Vector3.at(0.5, 0.5, 0.5);
```

Comment:
should be made static, it's already uppercase, and we don't have to allocate a Vector for each instance of the environment (especially due to potential clones)